ios/firebase_app_id_file.json

```diff
@@ -0,0 +1,7 @@
+{
+  "file_generated_by": "FlutterFire CLI",
```

Comment:
Add this file to your .gitignore file
Because that's related to your firebase project.